src/ServerRequestFactory.php

```diff
@@ -227,7 +226,6 @@ public static function marshalHeaders(array $server)
      * @param array $server
      * @param MessageInterface $request
      * @return Uri
-     * @deprecated as of 0.7.0; use marshalUriFromServer() instead.
```

Comment:
Okay, I’m too lazy to revert all the space changes, maybe it’s easier for you to pick the two-three interesting changes?

Too bad to have deprecated methods in 1.0.0… :( BUT you can tag 2.0.0 as that’s the main advantage of splitting repositories (or let’s remove them and add a BC notice… this thing has been released yesterday, I hope nobody already use that in production and using deprecated features)